bundles/org.openhab.binding.tado/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -307,4 +308,12 @@
 		<state readOnly="true"/>
 	</channel-type>
 
+	<channel-type id="openWindowRemainingTime">
+		<item-type>Number:Time</item-type>
+		<label>Open Window Override Remaining Time</label>
```

Comment:
While i don't have a improvement, we need to have these labels 2-3 words and < 23 chars. 
With 5 words and 35 characters, this exceeds it with a serious amount. Hope you can think of something creative.

Reply:
Ok. Done.